impl/random/cheriot.h

```diff
@@ -0,0 +1,24 @@
+uint32_t rand_32();
+
+static int
+hydro_random_init(void)
+{
+    const char       ctx[hydro_hash_CONTEXTBYTES] = { 'h', 'y', 'd', 'r', 'o', 'P', 'R', 'G' };
+    hydro_hash_state st;
+    uint16_t         ebits = 0;
+
+    hydro_hash_init(&st, ctx, NULL);
+
+    while (ebits < 256) {
+        uint32_t r = rand_32();
+        
+        //delay(10);
```

Comment:
If this delay necessary, or is that leftover debugging code?

Reply:
Good spot - not needed, I'll take it out